src/components/inputs/MultilineTextInput.js

```diff
@@ -48,7 +48,7 @@ const MutlilineTextInput = React.forwardRef<Props, Instance>(({ style, ...rest }
       ref={ref}
       keyboardAppearance={keyboardAppearance}
       // `placeholder` is needed for Android to avoid slight increase in height when input is empty.
-      placeholder=""
+      // placeholder=""
```

Comment:
Could we remove this instead of commenting it out?

Reply:
yes, done